test/irmin-pack/test_pack.ml

```diff
@@ -682,4 +682,5 @@ let misc =
     ("layers", Layered.tests);
     ("inodes", Test_inode.tests);
     ("trees", Test_tree.tests);
+    ("hashes", Test_hashes.tests);
```

Comment:
Can we put these first? If these tests fail, I think others are likely to fail as a consequence.